src/cbc_sdk/base.py

```diff
@@ -773,6 +773,15 @@ def _join(self, join_cls, field_name):
 
         return self._cb.select(join_cls, field_value)
 
+    def to_json(self):
+        """
+        Return a json object of the response.
+
+        Returns:
+            Any: The response dictionary representation.
+        """
+        return self._info
```

Comment:
I'd kind of rather have it return `copy.deepcopy(self._info)` instead, so you het a clean copy that doesn't affect the internal state of the object if you try and change it.

Reply:
That's fair though make sure to do the same thing for any other to_json overrides

Reply:
I absolutely agree with that!